requirements/requirements.txt

```diff
@@ -62,7 +62,7 @@ djangorestframework==3.5.4
 dnspython==1.15.0
 docutils==0.14            # via botocore
 dropbox==9.3.0
-elasticsearch==1.9.0
+elasticsearch==6.4.0
```

Comment:
this version doesn't match the PR description. Are we upgrading directly to 7.2 or 6.4?

Reply:
I will change this, documentation on elasticsearch-py.readthedocs.io is not updated and I didn't notice that we have 7.0.2 version of this library on pypi.org